Jenkinsfile_CNP

```diff
@@ -178,13 +179,27 @@ withPipeline(type, product, component) {
     }
 
     afterSuccess('smoketest:preview') {
+        if (notificationTestsEnabled) {
+            runNotificationTests()
+        }
```

Comment:
maybe we want an else statement here to cut the run time?

Reply:
what would we do in the else branch?
currently, the tests only run if you add the run-notification-tests label, and you also have the option to only generate one or a few notifications rather than the whole set